src/workerd/api/util.c++

```diff
@@ -288,4 +290,17 @@ void maybeWarnIfNotText(jsg::Lock& js, kj::StringPtr str) {
           "\". The result will probably be corrupted. Consider "
           "checking the Content-Type header before interpreting entities as text."));
 }
+
+kj::String fastEncodeBase64Url(kj::ArrayPtr<const byte> bytes) {
+  if (KJ_UNLIKELY(bytes.size() == 0)) {
+    return {};
+  }
+  auto expected_length = simdutf::base64_length_from_binary(bytes.size(), simdutf::base64_url);
+  auto output = kj::heapArray<char>(expected_length + 1);
+  auto actual_length = simdutf::binary_to_base64(
+      bytes.asChars().begin(), bytes.size(), output.asChars().begin(), simdutf::base64_url);
+  output[actual_length] = '\0';
```

Comment:
FWIW, I noticed the old code had an assert that `expected` was close to `actual`.  But maybe this isn't crucial to recheck, if we trust simdutf to make the same guarantee with regard to `base64_length_from_binary()` and `binary_to_base64()`.